packages/components/pages/f-checkout/test/visual/f-checkout-authenticated.visual.desktop.spec.js

```diff
@@ -315,6 +315,17 @@ describe('f-checkout - Dine In - Authenticated - Desktop Visual Tests', () => {
         // Assert
         browser.percyScreenshot('f-checkout - Dine in - Authenticated - "Duplicate Order Warning" Modal', 'desktop');
     });
+
+    it('should display the two notes fields if there is two noteTypes.', () => {
+        // Arrange
+        checkout = new Checkout('organism', 'checkout-component');
+        checkout.withQuery('&knob-Service Type', 'collection')
+            .withQuery('&knob-Is User Logged In', true)
+            .withQuery('&knob-Is ASAP available', true)
+            .withQuery('$knob-Note%20types', 'get-notes-config-split');
```

Comment:
I think it handles spaces already
```suggestion
            .withQuery('$knob-Note types', 'get-notes-config-split');
```